lib/pbench/test/unit/server/test_endpoint_configure.py

```diff
@@ -41,10 +41,6 @@ def check_config(self, client, server_config, host, my_headers={}):
         expected_results = {
             "identification": f"Pbench server {server_config.COMMIT_ID}",
             "api": {
-                # API endpoints with trailing Flask parameters are marked with
-                # a trailing "/" here; for example, /datasets/mappings/
-                # corresponds to /datasets/mappings/<string:dataset_view>";
-                # see endpoint_configure.py for more detail.
```

Comment:
You really want to remove this comment?  It seems like a useful thing to point out (and, it's still true after this change, right?).

Reply:
Yeah, I pulled the trailing `/` back in at the last moment, after having justified the removal of this comment... and I didn't think to restore it. 😆